File: gapy/test/test_vpro/vpro_test.py
import gapy
import numpy as np

gapy.vpro.vpro_read_legacy()

gapy.vpro.vpro_write()
gapy.vpro.vpro_init(0)

gapy.vpro.vpro_read()

print('nexp    ',gapy.vpro.nexp)
print('bt_exp  ',gapy.vpro.bt_exp)
print('arho_exp',gapy.vpro.arho_exp)
print('ni      ',gapy.vpro.ni[:,0])

gapy.vpro.vpro_compute_derived()

print('bunit   ',gapy.vpro.bunit)

File: gapy/test/test_vpro/vpro_test.f90
program test

  use vpro
  use expro_locsim_interface

  implicit none

  real :: ipccw,btccw,a_meters

  call vpro_read_legacy
  call vpro_write

  call vpro_read('./')

  print '(a,i0)'            ,'nexp    ',expro_n_exp
  print '(a,1pe12.5)'       ,'bt_exp  ',expro_b_ref
  print '(a,1pe12.5)'       ,'arho_exp',expro_arho
  print '(a,60(1pe12.5,1x))','ni      ',expro_ni(1,:)
  print '(a,60(1pe12.5,1x))','bunit   ',expro_bunit

  call expro_locsim_profiles('./',&
       -1,&
       0,&
       1,&
       0,&
       expro_n_ion+1,&
       0.5,&
       btccw,&
       ipccw,&
       a_meters)

  print '(a,1pe12.5)'       ,'kappa_loc',kappa_loc

end program test
